src/NumericalAlgorithms/LinearOperators/DefiniteIntegral.cpp

```diff
@@ -44,7 +43,9 @@ double definite_integral<1>(const DataVector& integrand,
   ASSERT(num_points == extents.product(),
          "num_points = " << num_points << ", product = " << extents.product());
   const DataVector& weights = Basis::lgl::quadrature_weights(num_points);
-  return ddot_(num_points, weights.data(), 1, integrand.data(), 1);
+  // The integral over the last remaining dimension is just a dot product
+  // between the integrand and the weights.
+  return (integrand, weights);
 }
 
 /// \cond
```

Comment:
Finding the spin requires solving an eigenvector problem with dggev_ (see https://github.com/sxs-collaboration/spec/blob/master/SurfaceFinder/StrahlkorperItems/StrahlkorperAKVDimensionfulSpin.cpp). Add a wrapper for that call here as well?

Reply:
@geoffrey4444 Blaze provides `blaze::eigen` wrappers over the LAPACK functions, is this what you are looking for? As part of our `Matrix` implementation we should wrap these too.

Reply:
Nope, @nilsleiffischer. I need to solve a generalized eigenvalue problem: A x = lambda B x, where A and B are matrices. Blaze currently only supports this for the case B=identity matrix. There is a lapack function for this, dggev, but it's not wrapped in Blaze.

Reply:
Oh right, you said so earlier. Perhaps we can get that into Blaze at some point.